### PR TITLE
Implemented player items methods

### PR DESCRIPTION
**Pull request by JJendryka**
Changes methods `add_items` and `remove_items` to `change_items`. Implements `change_items` and `has_items` in Player class

### CODE REVIEW

File: src/player.py [hunk 1]
```diff
@@ -27,17 +27,19 @@ def add_to_trait(self, name, value):
         """Add 'value' to 'trait'"""
         pass
 
-    def add_items(self, items):
+    def change_items(self, items):
         """Add all items in 'items' (dictionary name to value) in the player's inventory"""
-        pass
+        for item, count in items.items():
+            self.items[item] = self.items.get(item, 0) + count
```
Comment (Hazurl) [inline src/player.py]: I think `max(self.items.get(item, 0) + count, 0)` is better

**Review by Hazurl — COMMENTED** [review]
Everything else is good for me.

**Comment by Hazurl** [comment]
[Implement items related methods in Player class](https://trello.com/c/6B1kP2PH/27-implement-items-related-methods-in-player-class)

File: src/player.py [hunk 2]
```diff
@@ -27,17 +27,17 @@ def add_to_trait(self, name, value):
         """Add 'value' to 'trait'"""
         pass
 
-    def add_items(self, items):
+    def change_items(self, items):
         """Add all items in 'items' (dictionary name to value) in the player's inventory"""
-        pass
+        for item, count in items.items():
+            self.items[item] = max(self.items.get(item, 0) + count, 0)
```
Comment (Jackojc) [inline src/player.py]: just make a quick comment to say "this will clamp the value to 0 if .get returns a negative value."

File: src/player.py [hunk 3]
```diff
@@ -27,17 +27,17 @@ def add_to_trait(self, name, value):
         """Add 'value' to 'trait'"""
         pass
 
-    def add_items(self, items):
+    def change_items(self, items):
         """Add all items in 'items' (dictionary name to value) in the player's inventory"""
```
Comment (Jackojc) [inline src/player.py]: - i think function name would make more sense as `update_items`
- maybe add type hints like `items: dict`
- docstring is phrased a bit weirdly. try something like: "Update the players inventory to reflect the parameter `items`."

**Review by Hazurl — APPROVED** [review]

File: src/player.py [hunk 4]
```diff
@@ -4,7 +4,7 @@ class Player(object):
     """Describe the player, his inventory, traits, position and history"""
 
     def __init__(self, state):
-        pass
```
Comment (Jackojc) [inline src/player.py]: state = None
self.state = state

Comment (Hazurl) [reply src/player.py]: If you merge I don't think you need to recommit this change

**Comment by Hazurl** [comment]
Why github still say there is conflicts...

**Pull request merged**